src/main.js

```diff
@@ -4,7 +4,11 @@ import App from './App.vue';
 import RestClient from './rest';
 
 Vue.use(Girder);
-const girderRest = new RestClient({ apiRoot: 'http://localhost:8080/api/v1' });
+const girderRest = new RestClient({
+  apiRoot: process.env.NODE_ENV === 'production'
+    ? 'https://data.kitware.com/api/v1'
+    : 'http://localhost:8080/api/v1',
+});
```

Comment:
We could be more rigorous about this with `DefinePlugin` but this seems fine for a demo site.